components/observations/ObservationDetailView.tsx

```diff
@@ -216,34 +217,84 @@ export const ObservationCard: React.FunctionComponent<{
   }, [postHog, observation.center_id, observation.id]);
   useFocusEffect(recordAnalytics);
 
+// route.path will have the link we would put into the share button here
+const route = useRoute(); 
+console.log(route);
+
+// currently the back button will leave to the list of obs of your current default center
+// even if someone shares an observation with you from a different center
+// the header will still show the current center logo (that likely needs to change as design actually said to replace that with the share icon)
+// to open in expo: *check url and port, and correct obs ID that exists
+// example: npx uri-scheme open exp://192.168.1.8:8082/--/observation/866b81db-52b3-4f94-890c-0cae8f162097 --android
+const url = 'https://' + observation.center_id + '.us/observations/#/view/' + route.name + 's/' + route.params.id
```

Comment:
not sure if this info already exists, but we would need urls for all centers --if it doesnt exists we can store the url and get the proper one by using the center_id to query for it